src/ensembl/src/shared/helpers/formatters/geneFormatter.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export function getGeneName(geneName: string | null) {
+  if(geneName && geneName != 'novel transcript') {
```

Comment:
`!=` -> `!==`

I believe we have an eslint setting that enforces using strict equality comparison (triple equals) rather than equality comparison after typecasting (double equals). Strange that it didn't complain here.